src/main/kotlin/three/consulting/epoc/service/TimesheetEntryService.kt

```diff
@@ -46,8 +46,10 @@ class TimesheetEntryService(private val timesheetEntryRepository: TimesheetEntry
             timesheetEntryRepository.findAllByDates(startDate, endDate)
         }
 
+        val dateComparator = Comparator { entry1: TimesheetEntry, entry2: TimesheetEntry -> entry1.date.compareTo(entry2.date) }
+
         var columnNames = "hours;task;project;date;email\n"
-        val rows = entries.joinToString("") { entry -> "${entry.quantity};${entry.task.name};${entry.timesheet.project.name};${entry.date};${entry.timesheet.employee.email}\n" }
+        val rows = entries.sortedWith(dateComparator).joinToString("") { entry -> "${entry.quantity};${entry.task.name};${entry.timesheet.project.name};${entry.date};${entry.timesheet.employee.email}\n" }
```

Comment:
Using comparator is a more java thing where proper scope functions aren't available like in kotlin. I think you could achieve this same just by
```kotlin
entries.sortedBy{ it.date }
```
because `java.time.LocalDate` has proper comparators built in it.

Reply:
Okay. Good to know!